stdlib/2and3/shutil.pyi

```diff
@@ -130,10 +130,18 @@ if sys.version_info >= (3, 8):
 elif sys.version_info >= (3,):
     def which(cmd: _Path, mode: int = ..., path: Optional[_Path] = ...) -> Optional[str]: ...
 
-def make_archive(base_name: _AnyStr, format: str, root_dir: Optional[_Path] = ...,
-                 base_dir: Optional[_Path] = ..., verbose: bool = ...,
-                 dry_run: bool = ..., owner: Optional[str] = ..., group: Optional[str] = ...,
-                 logger: Optional[Any] = ...) -> _AnyStr: ...
+if sys.version_info >= (3, 8):
+    def make_archive(base_name: _Path, format: str, root_dir: Optional[_Path] = ...,
```

Comment:
I'm not sure whether I should use _Path or _AnyPath

Reply:
You'd need to use `_AnyPath` if e.g., you required multiple parameters to be the same type, or the return type to be the same type as a parameter.

From a quick scan, this doesn't seem to be the case here.